spec/support/utils/exercise-testing-util.js

```diff
@@ -24,6 +25,8 @@ const generateRandomExercise = (userId, exerciseId) => {
     + `\"progression\":\"${progression}\",`
     + `\"link\":\"${link}\",`
     + `\"rest_interval\":\"${rest_interval}\"}`
+    + `\"rest_interval_metric\":\"${rest_interval_metric}\"}`
+
```

Comment:
There should be a console.log under this to test it. Make sure this is the correct json because I see } twice which is usually not correct